guides/runtime-parser.md

```diff
@@ -0,0 +1,308 @@
+# Introduction
+
+*This guide assumes that the [App Shell guide](https://mobile.angular.io/guides/app-shell.html) has been completed.*
+
+Now we're going to take a look at another part of the Angular Mobile Toolkit - the App Shell Runtime Parser. The Runtime Parser is a library which has the following features:
+
+- Automatically generates the App Shell of our application by using a predefined template.
+- Provides the App Shell template from the local cache automatically for each view that should use it.
+
+This way, when the user navigates to a different view in the application she will instantly get a minimal working UI that will be later replaced with the view's full functionality. You can find more about the App Shell concept in the context of [Progressive Web Applications](https://developers.google.com/web/progressive-web-apps/) (PWA) on the [following link](https://developers.google.com/web/updates/2015/11/app-shell?hl=en).
+
+# Introduction to the Runtime Parser
+
+The Angular App Shell Runtime Parser works together with the App Shell directives, described in the [previous section](https://mobile.angular.io/guides/app-shell.html), and [Angular Universal](https://universal.angular.io).
+
+Now lets dig deeper in the features of the parser!
+
+First, lets suppose we have the following "Hello Mobile" component:
+
+```typescript
```

Comment:
Is this exactly the same code as the previous step? If so, say something like, "let's review...".